src/ImageCard.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class ImageCard extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      showResults: false
+    }
+    this.onClick = this.onClick.bind(this)
+  }
+ 
+  onClick(){
+    this.setState({
+      showResults: !this.state.showResults
+    })
+  }
+
+  render(){
+    let {iconSrc, iconDescription, content} = this.props
+    return (
+      <div className={`column column-block text-center combo card`} style={{marginBottom:0, paddingBottom: `25px`}}>
+        {
+          iconDescription && <h5 className="species-name">{iconDescription}</h5>
+        }
+
+        <span className={`species-icon`} style={{fontSize: `800%`}}>
+          <img className={`icon`} alt={iconDescription} src={iconSrc}/>
+        </span>
+
+        {
+          content &&
+            <ul className={`content`} style={{listStyle:`none`, paddingLeft:`0`, marginLeft:`0`}}>
+              {
+                content.map((item, idx) => { 
+                  return(
+                    idx < 5 || (idx >= 5 && this.state.showResults) ?
```

Comment:
You can simplify the conditional rendering by mapping all `content` elements and then use `slice` depending only on `this.state.showResults` (see comment above).